src/lxc/execute.c

```diff
@@ -111,16 +111,15 @@ static struct lxc_operations execute_start_ops = {
 };
 
 int lxc_execute(const char *name, char *const argv[], int quiet,
```

Comment:
Note, this change prevents old `lxc-execute` binaries to be useable with `liblxc`'s that have a new version of `lxc_execute()`. This is fine since `lxc-execute` used internal symbols which is something we won't support in the future anymore. Also, we never intended to support old binaries + new `liblxc`.